spec/board_spec.rb

```diff
@@ -19,15 +19,23 @@
   it 'has sixteen key-value pairs' do
     board = Board.new
 
-    #board_size = board.size
     expect(board.cells.size).to eq(16)
   end
 
   it 'has valid coordinates' do
     board = Board.new
 
     expect(board.valid_coordinate?('A1')).to eq(true)
+    expect(board.valid_coordinate?('B4')).to eq(true)
+    expect(board.valid_coordinate?('C4')).to eq(true)
   end
 
+  it 'if provided invalid coordinate, returns false' do
+    board = Board.new
+
```

Comment:
These look great. Adding multiple arguments to return false to ensure our method is working was a great idea!